### PR TITLE
AMP-69336 Make identify example concise

### PR DESCRIPTION
**Pull request by jooohhn**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/guides/segment-migration-guide.md [hunk 1]
```diff
@@ -82,10 +82,7 @@ Both Segment and Amplitude SDKs are meant to capture first party data by trackin
     <td>
     ```typescript
     setUserId('12091906-01011992');
-    const identifyObj = new Identify();
-    identifyObj.set('name', 'Grace Hopper');
-    identifyObj.set('email', 'grace@usnavy.gov');
-    identify(identifyObj);
+    identify(Identify().set('name', 'Grace Hopper').set('email', 'grace@usnavy.gov'))
```
Comment (justin-fiedler) [inline docs/guides/segment-migration-guide.md:85]: Looks better. What do you think of this format to make each `set` more clear?

```
identify(Identify()
  .set('name', 'Grace Hopper')
  .set('email', 'grace@usnavy.gov')
)
```

**Review by justin-fiedler — CHANGES_REQUESTED** [review]
We should update GroupIdentify usage examples as well.

**Review by ppatibandla — COMMENTED** [review]
John, can you please review and make iOS & Android  identify & group identify calls concise as well.

File: docs/guides/segment-migration-guide.md [hunk 2]
```diff
@@ -153,8 +153,8 @@ Both Segment and Amplitude SDKs are meant to capture first party data by trackin
     ```swift
     Amplitude.instance().setUserId("abc")
     let identify = AMPIdentify()
-        .set("email", value: "female")
-        .set("age",value: NSNumber(value: 20))
+      .set("email", value: "female")
+      .set("age",value: NSNumber(value: 20))
     Amplitude.instance().identify(identify)
```
Comment (justin-fiedler) [inline docs/guides/segment-migration-guide.md:158]: What about making this all one line? 

```
    Amplitude.instance().identify(AMPIdentify()
      .set("email", value: "female")
      .set("age",value: NSNumber(value: 20))
    )
```

Comment (jooohhn) [reply docs/guides/segment-migration-guide.md:158]: @justin-fiedler I noticed our SDKs don't follow the same format for identify. Should we update them too?


- [Browser](https://www.docs.developers.amplitude.com/data/sdks/typescript-browser/#identifyset) 
- [iOS](https://www.docs.developers.amplitude.com/data/sdks/ios-swift/#identify)

**Comment by jooohhn** [comment]
@justin-fiedler sorry about that, style should be consistent for all the languages now

**Review by justin-fiedler — APPROVED** [review]

File: docs/guides/segment-migration-guide.md (hunk 2 above)
Comment (justin-fiedler) [reply docs/guides/segment-migration-guide.md]: Good question @jooohhn , let me ask the team if the verbose usage was intentional there. We can update in another PR.

**Pull request merged**